packages/taquito/src/subscribe/polling-subcribe-provider.ts

```diff
@@ -0,0 +1,135 @@
+import { BlockResponse } from '@taquito/rpc';
+import { BehaviorSubject, from, Observable, ObservableInput, OperatorFunction, timer } from 'rxjs';
+import {
+  concatMap,
+  distinctUntilKeyChanged,
+  first,
+  pluck,
+  publish,
+  refCount,
+  retry,
+  switchMap,
+} from 'rxjs/operators';
+import { Context } from '../context';
+import { evaluateFilter } from './filters';
+import { Filter, SubscribeProvider, Subscription, OperationContent } from './interface';
+import { ObservableSubscription } from './observable-subscription';
+import BigNumber from 'bignumber.js';
+
+export interface PollingSubscribeProviderConfig {
+  streamerPollingIntervalMilliseconds?: number;
```

Comment:
I wonder if these names in the interface can be shortened. Since it's already a property of `PollingSubscribeProviderConfig`, can these names possibly be shortened?

Reply:
Yes I can change streamerPollingIntervalMilliseconds for pollingIntervalMilliseconds